src/validation/requestSchema.js

```diff
@@ -11,6 +11,17 @@ const requestTripSchema = Joi.object({
   accommodation: JoiValidator.validateString().required(),
 });
 
+const searchRequestTripSchema = Joi.object({
```

Comment:
Please make the schema require at least one key. Check out this [link](https://github.com/hapijs/joi/blob/master/API.md#objectminlimit) if you don't know how.

Reply:
Okay cool, thank you

Reply:
done